src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -2050,10 +2064,13 @@ public DType GetUsedScopeFields(CallInfo call)
 
             foreach (var name in GetLambdaParamNames(call.ScopeNest + 1))
             {
-                var fError = false;
+                var fError = false;
+
+                call.CursorType.TryGetLogicalName(name.Name, out var fieldName);

```

Comment:
>fieldName [](http://example.com/codeflow?start=69&length=9)

In a TryGetXXX method, if it returns false, then the out parameter is null. Will that cause a problem when this value is used?